docs/get-started/running-nodes.mdx

```diff
@@ -4,306 +4,218 @@ title: 'Running Etherlink Nodes'
-  "unsafe-pvm-patches": [
-    { "increase_max_nb_tick": "50_000_000_000_000" }
-  ],
-  "pre-images-endpoint": "https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0",
-  "mode": "observer" }
-```
-
-With this configuration file, it is now possible to start the Smart Rollup node
-in observer mode. Note that you will need to be able to access to the RPCs of
-an Octez node, running in archive mode. For testing purposes, you can rely
-on public endpoints like the one provided by TzKT for instance.
-
-```bash
-# for testing purposes
-octez-smart-rollup-node --endpoint https://rpc.tzkt.io/mainnet run \
-    --data-dir $sr_observer_data_dir \
-    --apply-unsafe-patches
-```
-
-A few remarks.
-
-- `--apply-unsafe-patches` is only required the first time you start your node,
-  and can be removed from the command-line afterwards. Similarly, the
-  `unsafe-pvm-patches` field can be removed from the configuration field.
-- Once the Smart Rollup node has caught up, you can safely connect it to a
-  rolling `octez-node`.
-
-Once you have the rollup node running, you can confirm that you are able query
-it; for instance by requesting the latest read tezos block:
-
-```bash
-curl -s http://localhost:8932/global/block/head
-```
-
-### Running `octez-evm-node` in Observer Mode
-
-### From Genesis
-
-In order to run an `octez-evm-node` in observer mode from Genesis, you need:
-
-- An access to a Smart Rollup node tracking the state of Etherlink (see the
-  previous section to setup one yourself, which is probably necessary until
-  public nodes become available).
-- The `installer.hex` file that you can get by following the “Building
-  Etherlink’s Initial Kernel” appendix.
-
-```bash
-octez-evm-node init config --devmode \
-    --data-dir $evm_observer_dir --rollup-node-endpoint $sr_node_observer_rpc \
-    --preimages-endpoint https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0 \
-    --evm-node-endpoint https://node.mainnet.etherlink.com \
-    --initial-kernel observer/installer.hex \
-    --force
-```
-
-If you do not want to relies on a public preimages endpoint (though it is worth
-mentioning again it is safe), you can move the `wasm_2_0_0/` directory built
-along the `installer.hex` file in the directory `$evm_observer_dir`. Note that
-this means you will have to provision this directory with the preimages of
-every kernel voted by the community and deployed on Etherlink after that.
-
-Starting the node is as simple as running this command.
-
-```bash
-octez-evm-node run observer --data-dir $evm_observer_node
-```
-
-By default, the `octez-evm-node` exposes its JSON RPC API endpoint to `localhost:8545`.
-You can test that everything works as expected by running some RPC requests manually or by
-setting your wallet to use your local node.
+## Running Etherlink nodes
+
+Etherlink nodes require access to the RPCs of a running Smart Rollup node.
+Therefore, to run an Etherlink node, you must also run a Smart Rollup node.
```

Comment:
This is not true, you could uses a public rollup node endpoint. There is none for the moment but I think it could be interesting to have one.